filebeat/module/elasticsearch/gc/manifest.yml

```diff
@@ -0,0 +1,11 @@
+module_version: 1.0
+
+var:
+  - name: paths
+    default:
+      - /var/log/elasticsearch/gc.log.[0-9]*
```

Comment:
What is the name of the first log? If it is called `gc.log` I think the pattern will not match it.

Reply:
Got it, then the pattern makes sense.